atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolImpl.java

```diff
@@ -312,7 +312,7 @@ private void setServersInPoolTo(Set<InetSocketAddress> desiredServers) {
             cassandra.refreshTokenRangesAndGetServers();
         }
 
-        log.debug("Cassandra pool refresh added hosts {}, removed hosts {}.",
+        log.info("Cassandra pool refresh added hosts {}, removed hosts {}.",
```

Comment:
nit: How valuable is an empty refresh line? Maybe we can have a refresh that did something at INFO and a refresh that did nothing at DEBUG.

Reply:
I like it.